src/sqlancer/stonedb/StoneDBOptions.java

```diff
@@ -30,6 +31,12 @@ public class StoneDBOptions implements DBMSSpecificOptions<StoneDBOracleFactory>
     public List<StoneDBOracleFactory> oracles = List.of(StoneDBOracleFactory.NOREC);
 
     public enum StoneDBOracleFactory implements OracleFactory<StoneDBGlobalState> {
+        Fuzz {
```

Comment:
Perhaps this could be `FUZZER`, so we use the same name as the other implementations?

Reply:
sure, will change